server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -68,6 +71,7 @@ export class TenantManager {
 		private readonly cache?: ICache,
 	) {
 		this.isCacheEnabled = this.cache ? true : false;
+		this.incomingKeyVersion = EncryptionKeyVersion.key2023;
```

Comment:
incomingKeyVersion should be an abstract function. Different teams should override this function to provide their own key name to use. Hardcoding this value over here may break it for a team who is using a different value and has not rotated its key.

Reply:
As far as I know, we are the only team who consumes r11s-base package. Given the tight schedule for kek roataion,  I think we can keep it for now and I will make it configable some time later?

Reply:
I agree with Manish here. We can't assume we are the only ones using the r11s. Since this code is in OSS, let's make sure it is backward compatible.

Reply:
Sounds good, I will make it a config, and it will be read into a variable into SecretManager, and assume it could be empty.